internal/cli/app_docs.go

```diff
@@ -233,13 +233,34 @@ func endingSpace(w io.Writer, i, tot int) {
 
 func splitFields(fields []*docs.FieldDocs) (required, optional []*docs.FieldDocs) {
 	var o, r []*docs.FieldDocs
-
+	not_req := true
```

Comment:
We use camelcase, not snakecase for variable names.
```suggestion
	notRequired := true
```